Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -2067,4 +2072,31 @@ namespace O3DE::ProjectManager
     {
         m_pythonErrorStrings.push_back(errorString);
     }
+
+    bool PythonBindings::ExtendSysPath(const AZStd::vector<AZStd::string>& extendPaths)
+    {
+        AZStd::unordered_set<AZStd::string> oldPathSet;
```

Comment:
I recommending making the oldPathSet a set of AZ::IO::Path.
That would take care of any quirks in path comparison when running the `contains` function below.
```suggestion
        AZStd::unordered_set<AZ::IO::Path> oldPathSet;
```